integration/monai/examples/spleen_ct_segmentation_local/README.md

```diff
@@ -225,3 +233,17 @@ Then, submit the job to run FedAvg with HE:
 ```
 nvflare job submit -j jobs/spleen_ct_segementation_he
 ```
+
+### 5.5 MLflow experiment tracking results
+
+If using the MLflow tracking server, access the MLflow dashboard in your browser using the default tracking uri `http://127.0.0.1:5000`.
+
+Otherwise, view the results by running the following command at the `mlruns/` directory in the server workspace:
+
+```
+mlflow ui --port 5000
+```
+
+Once the training is started, you can see the experiment curves for the local clients in the current run on the MLflow dashboard.
```

Comment:
more clarification on this:

`Otherwise, view the results by running the following command in the directory that contains the `mlruns/` directory in the server workspace:`